inst/codec_data/parcel/parcel.R

```diff
@@ -0,0 +1,101 @@
+if (tryCatch(read.dcf("DESCRIPTION")[1, "Package"] == "codec", finally = FALSE)) {
+  devtools::load_all()
+} else {
+  library(codec)
+}
+message("Using CoDEC, version ", packageVersion("codec"))
+library(dplyr)
+library(sf)
+
+cagis_parcels <- dpkg::stow("https://github.com/geomarker-io/parcel/releases/download/cagis_parcels-v1.1.0/cagis_parcels-v1.1.0.parquet") |>
+  dpkg::read_dpkg() |>
+  tibble::as_tibble() |>
+  select(parcel_id, centroid_lat, centroid_lon, land_use, condo_id, market_total_value, acreage, homestead, rental_registration)
+
+online_parcels <- dpkg::stow("https://github.com/geomarker-io/parcel/releases/download/auditor_online_parcels-v0.2.0/auditor_online_parcels-v0.2.0.parquet") |>
+  dpkg::read_dpkg() |>
+  tibble::as_tibble()
+
+property_code_enforcements <- dpkg::stow("https://github.com/geomarker-io/parcel/releases/download/property_code_enforcements-v1.0.0/property_code_enforcements-v1.0.0.parquet") |>
+  dpkg::read_dpkg() |>
+  tibble::as_tibble() |>
```

Comment:
with the newer version of dpkg, converting to tibble before using dplyr shouldn't be needed; I just wanted to make sure it was working as expected...